src/utils/issue.ts

```diff
@@ -1,26 +1,32 @@
 import { Context } from "../types/context";
-import { Issue, ISSUE_TYPE, PullRequest, Review } from "../types/payload";
+import { GitHubIssueSearch, Review } from "../types/payload";
 import { getLinkedPullRequests, GetLinkedResults } from "./get-linked-prs";
 
 export function isParentIssue(body: string) {
   const parentPattern = /-\s+\[( |x)\]\s+#\d+/;
   return body.match(parentPattern);
 }
 
-export async function getAssignedIssues(context: Context, username: string): Promise<Issue[]> {
+export async function getAssignedIssues(context: Context, username: string): Promise<GitHubIssueSearch["items"]> {
   const payload = context.payload;
 
   try {
-    return await context.octokit.paginate(
-      context.octokit.issues.listForRepo,
-      {
-        owner: payload.repository.owner.login,
-        repo: payload.repository.name,
-        state: ISSUE_TYPE.OPEN,
+    return await context.octokit
+      .paginate(context.octokit.search.issuesAndPullRequests, {
+        q: `org:${payload.repository.owner.login} assignee:${username} is:open`,
         per_page: 100,
-      },
-      ({ data: issues }) => issues.filter((issue: Issue) => !issue.pull_request && issue.assignee && issue.assignee.login === username)
-    );
+        order: "desc",
+        sort: "created",
+      })
+      .then((issues) =>
+        issues.filter((issue) => {
```

Comment:
Is the filter needed since the search specifies `is:open`? Might wanna add `is:pr` as well to the query.

Reply:
> Is the filter needed since the search specifies is:open

no I guess not

> Might wanna add is:pr as well to the query.

This function is looking for issues not PRs so if anything `is:issue` should be added no, can we do `is:issue` yeah?

Reply:
I think it is redundant and error prone if we ever change the query later, since the same check is ran twice. Hopefully the API should be reliable.

I missed the `!` you are right, `is:issue` would be the right call.